drivers/can/can_rcar.c

```diff
@@ -1056,6 +1092,9 @@ static const struct can_driver_api can_rcar_driver_api = {
 		.phase_seg2 = DT_INST_PROP_OR(n, phase_seg2, 0),		\
 		.sample_point = DT_INST_PROP_OR(n, sample_point, 0),		\
 		.pcfg = PINCTRL_DT_INST_DEV_CONFIG_GET(n),			\
+		.phy = DEVICE_DT_GET_OR_NULL(DT_INST_PHANDLE(n, phys)),		\
+		.max_bitrate =							\
+			CAN_TRANSCEIVER_DT_MAX_BITRATE_INST_GET(n, 1000000),	\
```

Comment:
Wouldn't it be a more transparent solution to check if we are in a CAN FD mode of operation directly in the macro as soon as we can deduct the max bitrate from it ? (1Mbps for CAN and 5Mbps for CANFD)

Reply:
Some CAN controllers may impose other restrictions on the maximum bitrate than the maximum bitrates supported by the standard (e.g. some CAN-FD controllers only support 2Mbit/s). This is why I left it up to the individual driver to specify its maximum bitrate.